docs/UserGuide.md

```diff
@@ -72,30 +69,29 @@ Graphical User Interface (GUI) | User interface that allows users to interact vi
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Supported platforms:**<br>
-PropertyWhiz has been extensively tested on Microsoft Windows 10 and Ubuntu 20.04. However, recent versions of Windows and GNU Linux are also likely to work.
+PropertyWhiz has been extensively tested on Microsoft Windows 10 and Ubuntu 20.04. However, recent versions of Windows and GNU Linux are also likely to work with PropertyWhiz.
```

Comment:
Not quite sure if we should throw "big words" such as GNU Linux in Quick Start section... Maybe it can be removed?